mtp_api/apps/prison/models.py

```diff
@@ -7,5 +7,5 @@ class Prison(TimeStampedModel):
     nomis_id = models.CharField(max_length=3, primary_key=True)
     name = models.CharField(max_length=500)
 
-    def __unicode__(self):
+    def __str__(self):
```

Comment:
Why `__str__()` instead of `__unicode__()`? I thought the latter was preferred these days?